### ISSUE TITLE
libnfc.driver.pn53x_usb	Unable to write to USB

### ISSUE THREAD

**Issue by GoogleCodeExporter** [timmerk/libnfc#223]
```
What steps will reproduce the problem?
1. Compile libnfc > 1.7
2. Run any of the utils with snapper feeder (pn531 chipset)

What is the expected output? What do you see instead?
When running nfc-list or any of the other utils (nfc-list is my example) I get 
the following error:
libnfc.driver.pn53x_usb Unable to write to USB (Connection timed out)

What version of the product are you using? On what operating system?
I am running the development version from the git repo 
(libnfc-1.7.0-rc2-18-gb675db7) on backtrack 5

Please provide any additional information below.

Using version 1.6 doesnt cause this problem, however the tool (mfcuk) I am 
trying to compile requires the latest version. I have tried all of the rc's for 
1.7
```

Original issue reported on code.google.com by `Tred...@gmail.com` on 29 Jan 2013 at 11:07

**Comment by GoogleCodeExporter** [comment]
```
PN531 demo board tested successfully with libnfc-1.7.0-rc2

Are you sure nothing is keeping the device busy?
pcscd?
some kernel module such as pn533?

could you run it with debug and show the trace?
LIBNFC_LOG_LEVEL=3 nfc-list


```

Original comment by `yob...@gmail.com` on 29 Jan 2013 at 11:29

**Comment by GoogleCodeExporter** [comment]
```
and also bottom messages of "dmesg", if anything relevant
```

Original comment by `yob...@gmail.com` on 29 Jan 2013 at 11:30

**Comment by GoogleCodeExporter** [comment]
```
I think I got it!

```

Original comment by `romu...@libnfc.org` on 29 Jan 2013 at 11:55
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 98948172b40e.
```

Original comment by `romu...@libnfc.org` on 29 Jan 2013 at 11:58
- Changed state: **Fixed**
